### PR TITLE
fix etcd v3 watch dir bug

### PR DESCRIPTION
**Pull request by DoubleSpout**
fix etcd v3 watch dir bug
add watch key="" empty case, mean watch all key change

### CODE REVIEW

**Comment by membphis** [comment]
@nic-chen please take a look at this PR

File: lib/resty/etcd/v3.lua [hunk 1]
```diff
@@ -291,10 +292,11 @@ local function set(self, key, val, attr)
 end
 
 local function get(self, key, attr)
+
     -- verify key
-    key = utils.normalize(key)
-    if not key or key == '/' then
-        return nil, "key invalid"
+    local _, err = utils.verify_key(key)
```
Comment (membphis) [inline lib/resty/etcd/v3.lua:297]: we need to add test case for this changing

Comment (DoubleSpout) [reply lib/resty/etcd/v3.lua:297]: i have added `etcdIns:get` and `etcdIns:set` empty key's test case

**Comment by membphis** [comment]
@DoubleSpout many thx, merged

**Pull request merged**